cmd/ping.go

```diff
@@ -46,12 +42,20 @@ var pingFlags = []cli.Flag{
 	cli.IntFlag{
 		Name:  "count, c",
 		Usage: "perform liveliness check for count number of times",
-		Value: 4,
 	},
-	cli.DurationFlag{
+	cli.IntFlag{
+		Name:  "error-count, e",
+		Usage: "exit if errors more than consecutive error count",
+		Value: 50,
```

Comment:
I don't think this should have a default value, I think it is okay to not activate error-count by default